src/server/server.rs

```diff
@@ -75,6 +76,7 @@ impl<T: RaftStoreRouter<RocksEngine> + Unpin, S: StoreAddrResolver + 'static> Se
         security_mgr: &Arc<SecurityManager>,
         storage: Storage<E, L>,
         cop: Endpoint<E>,
+        coprv2: CoprV2Endpoint<E>,
```

Comment:
Would it be better if it is `copr_v2: coprocessor_v2::Endpoint<E>`?

Reply:
Yes, good idea! Then the `Endpoint` struct name matches the filename. Changed it accordingly to `coprv2::Endpoint`, because the module is still called `coprv2`.

Reply:
Ahh, I see. Then why doesn't it named `coprocessor_v2` aligning with `coprocessor`

Reply:
Sure thing! I thought we said that we name the new module `coprv2`, but I like `coprocessor_v2` more anyway :)